### ISSUE TITLE
a11y concerns

### ISSUE THREAD

**Issue by ChrisDeHaan** [ChrisDeHaan/Personal-Portfolio#11]
all images should have alt tags and color variations should fall within WCAG AA guidelines.

**Comment by ChrisDeHaan** [comment]
alt tags were placed correctly according to tutor. closing issue now
